docs/intro.md

```diff
@@ -32,7 +32,7 @@ Differentiating features between popular vendors' tools and Mojito out of the bo
 Feature | Optimizely X | Google Optimize | Mojito
 --- | --- | --- | ---
 Open-source license | ❌ | ❌ | ✅ BSD3
-Light front-end codebase * | ❌~80kb | ❗~25kb | ✅<5kb
+Light front-end codebase | ❌~80kb * | ❗~25kb * | ✅<6kb ^
```

Comment:
For consistency, should this be `~5.5kb`

Reply:
Good point, might as well hey. Just updated this.